### DISCUSSION TITLE
A [FR] media filter, based on music provider

### DISCUSSION THREAD

**Discussion by erkr** [music-assistant/support#1538] [Feature requests and ideas]
I would like to have the option to filter media in the albums, artist, tracks and playlist views. 
Maybe more filter ideas pop up, but I have the wish to filter on the music providers that offer particular media. 
E.g show only the local playlist, or show only playlist of a specific Spotify account.

**Comment by OzGav** [comment]
On Discord a guy was looking at pulling genre information in. Marcel has said before that genre filtering and use of genres to populate “radio” playlists will come after 2.0 is shipped. 

Related to https://github.com/orgs/music-assistant/discussions/562

**Comment by erkr** [comment]
@OzGav you are right, those are other filter examples.
I don't expect this feature to be part of the initial v2 release😉

**Comment by OzGav** [comment]
This has been implemended in the current nightly and will be available in 2.7.0 beta 24 and later

**Reply by X-dark** [reply]
This is great. ~~Could it be done multi-choice selection, as currently we can either view all or one provider?~~

_edit: it is, just missed it as there was no hint from the UI_

**Comment by OzGav** [comment]
The feature request has been implemented successfully. As such this feature request will be closed and locked. If there are enhancements desired then open a new [feature request](https://github.com/orgs/music-assistant/discussions/categories/feature-requests-and-ideas). If there is a question about how this works then check the [documentation](https://beta.music-assistant.io/), or ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord. If there are problems then ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord or open an [issue](https://github.com/music-assistant/support/issues). Thanks for your interest in seeing this idea being brought to life.
